enhancements/20230504-freeze-fs-during-snapshot-creation.md

```diff
@@ -0,0 +1,69 @@
+# Freeze FS during snapshot creation
+
+## Summary
+
+HEP introduces proposed changes needed to harvester to meet the requirements of performing a fs freeze on guests when a backup is performed.
+
+Currently when a user initiates a VM backup, no guest level action is performed to ensure the filesystems are sync or io is paused.
+
+If a backup is triggered during a high IO event, the backup can be inconsistent.
+
+### Related Issues
+
+[GH issue 1723 ](https://github.com/harvester/harvester/issues/1723)
+
+### Goals
+
+- Attempt to perform fs freeze if qemu-guest-agent is installed in the guest when a backup is triggered
+
+## Proposal
+
+### User Stories
+
+#### Need for consistent backups for VM workloads
+#####  User Experience In Detail
+A VM backup initiated during times of high IO, can be inconsistent. This can be due to the fact the backup controller doesn't attempt to pause fs io when a backup is performed.
```

Comment:
```suggestion
A VM backup initiated during times of high I/O can be inconsistent. This can be due to the fact that the backup controller doesn't attempt to pause I/O when a backup is performed.
```